pandas/tests/frame/test_alter_axes.py

```diff
@@ -418,6 +418,14 @@ def test_rename(self):
                               pd.Index(['bar', 'foo'], name='name'))
         assert renamed.index.name == renamer.index.name
 
+    def test_rename_axis_inplace(self):
```

Comment:
can you also add a similar test in series/test_alter_axes.py

and tests this with axis=1 as well.